app/controllers/pages/questions_controller.rb

```diff
@@ -41,16 +35,9 @@ def edit
   end
 
   def update
-    page.load(page_params)
-    page.page_heading = draft_question.page_heading
-    page.guidance_markdown = draft_question.guidance_markdown
-    page.answer_type = draft_question.answer_type
-    page.answer_settings = draft_question.answer_settings
+    page.load(page_params_for_forms_api)
 
-    @question_form = Pages::QuestionForm.new(page_params.merge(answer_settings: draft_question.answer_settings,
-                                                               page_heading: draft_question.page_heading,
-                                                               guidance_markdown: draft_question.guidance_markdown,
-                                                               draft_question:))
+    @question_form = Pages::QuestionForm.new(page_params_for_form_object)
```

Comment:
How come this instance didn't use `answer_type` before? 🤔 Does it make a difference if it has it set now?

Reply:
Great question, I had a look into it and it doesn't make a difference.  I think I'm going to leave it as is for the moment so we dont risk breaking it now. When we come to adding "Check your question" page all of this will become a lot clearer as to what is needed for the form fields and what is needed for the page object to be created/updated.